.github/chainguard/lifecycle-cve-pr-closer.sts.yaml

```
@@ -1,8 +1,7 @@
 issuer: https://accounts.google.com
 
 # lc-cve-pr-closer-cron@prod-enforce-fabc.iam.gserviceaccount.com
-claim_pattern:
-  email: 'lc-cve-pr-closer-cron@prod-enforce-fabc\.iam\.gserviceaccount\.com'
+subject_pattern: "(101045395775641394837)"
```

```suggestion
subject: "101045395775641394837"
```

If we only expect one subject ever, we can dispense with the regex. Either way is fine though.

---

I think a single subject is expected. Updated!